binzx/otomi

```diff
@@ -207,6 +207,7 @@ vars=(
   AZURE_CLIENT_SECRET
   AZURE_TENANT_ID
   CI
+  CUSTOM_NETWORK
```

Comment:
this env is not used inside container therefore you do not need to mention it here